src/components/TableOfContents.astro

```diff
@@ -0,0 +1,60 @@
+---
+import type { MarkdownHeading } from "astro";
+import TableOfContentsHeading from "./TableOfContentsHeading.astro";
+import type { HeirarchicalHeading } from "../types";
+
+interface Props {
+  headings: MarkdownHeading[];
+}
+
+const { headings } = Astro.props;
+
+function buildSubheading(i: number): [number, HeirarchicalHeading[]] {
```

Comment:
What do you mean by "the top"? Do you mean the `Array.` part? I feel like we should just keep the same content rendered by grain doc

Reply:
> What do you mean by "the top"? Do you mean the `Array.` part? I feel like we should just keep the same content rendered by grain doc

Yeah I was referring to the `Array.` part, while I agree semantically that it makes sense to stay consistent with grainDoc, the `Array.Immutable.xxxx` seems to get cut off even on larger displays and I feel like by just chopping the prefix of `Array` it will help with that a lot.

Reply:
Should we just change the grain doc output to remove these?

Reply:
I don't know about that, the way I have been thinking of it is we are building a table of contents on the right hand side which makes the `Array.` part here redundant, and because of how much space it takes up I think it makes sense to chop it off, however if you were previewing the markdown directly or when viewing it inline having the full header is nice so you know exactly how to get to the function or value from the top level module. 

To be fair though as well we do change the header level for submodules so maybe it would be more correct to change this on the graindoc side.